pydantic_ai_slim/pydantic_ai/__init__.py

```diff
@@ -123,6 +126,8 @@
     'UserPromptNode',
     'capture_run_messages',
     'InstrumentationSettings',
+    # embeddings
+    'Embedder',
```

Comment:
was it intentional that you didn't include the other embedding types like `EmbeddingSettings` and `EmbeddingResult` here? I think the analogous agent/model types are exported here, at least some of them.